charts/tempo-distributed/Chart.yaml

```diff
@@ -31,3 +31,8 @@ dependencies:
     version: 0.2.2
     repository: https://grafana.github.io/helm-charts
     condition: metaMonitoring.grafanaAgent.installOperator
+  - name: rollout-operator
+    alias: rollout_operator
+    repository: https://grafana.github.io/helm-charts
+    version: 0.15.0
+    condition: rollout_operator.enabled
```

Comment:
This variable isn't defined in values, which for some reason enabled installation of rollout-operator by default.